kagami/database/database_service/engine.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+from ...config import SupervisorConfig
+
+
+class BaseSession:
+    _session: AsyncSession
+
```

Comment:
You didn't use BaseSession so why you wrote this?
Besides, Session don't need a base class to define. Unlike database models.